modules/bmo-editing-the-hostfirmwaresettings-resource.adoc

```diff
@@ -0,0 +1,39 @@
+// This is included in the following assemblies:
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="editing-the-hostfirmwaresettings-resource_{context}"]
+= Editing the HostFirmwareSettings resource
+
+You can edit the `HostFirmwareSettings` of provisioned hosts.
+
+.Procedure
+
+. Get the list of hosts:
++
+[source,terminal]
+----
+$ oc get hfs -n openshift-machine-api
+----
+
+. Edit a host's `HostFirmwareSettings` resource:
++
+[source,terminal]
+----
+$ oc edit hfs -n openshift-machine-api/<host_name>
+----
++
+Where `<host_name>` is the name of a provisioned host. The `HostFirmwareSettings` resource will open in the default editor for your terminal.
+
+. Add name/value pairs to the `spec.settings` section:
++
+.Example
+[source,terminal]
+----
+spec:
+  settings:
+    name: value
+----
```

Comment:
I believe here schema should be mentioned to receive available values. In addition it should be mantioned that chage will take place only for BMH in `provisioned` state and not `externally provisioned`